addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java

```diff
@@ -165,14 +165,6 @@ public ConfigMap createJenkinsConfigMap(OsioProjectile projectile, GitRepository
         return cm;
     }
 
-    public void triggerBuild(OsioProjectile projectile) {
-        //TODO remove this call (the trigger build should already done by the webhook) and change the countdown latch to 1 in OsioStatusClientEndpoint
-        String namespace = tenant.getDefaultUserNamespace().getName();
-        openShiftService.triggerBuild(projectile.getOpenShiftProjectName(), namespace);
-
-        projectile.getEventConsumer().accept(new StatusMessageEvent(projectile.getId(), OPENSHIFT_PIPELINE));
```

Comment:
@gastaldi If this is being removed, shouldn't the `OPENSHIFT_PIPELINE` event removed from 
https://github.com/fabric8-launcher/launcher-backend/blob/577011eaa5934b8897d5f7e722f37ff9f49ce393/addons/osio-addon/src/main/java/io/fabric8/launcher/osio/web/endpoints/OsioImportEndpoint.java#L63

and 

https://github.com/fabric8-launcher/launcher-backend/blob/577011eaa5934b8897d5f7e722f37ff9f49ce393/addons/osio-addon/src/main/java/io/fabric8/launcher/osio/web/endpoints/OsioLaunchEndpoint.java#L64

Reply:
@jarifibrahim the `OPENSHIFT_PIPELINE` event is sent when the createJenkinsConfigMap method is called

Reply:
Oh yes. My bad. Thanks @gastaldi .

On a side note - why are we sending two events with the same name?

Reply:
I wonder about that too. Probably that was before the StatusEventKind was turned into an interface.

Reply:
I wonder about that too. Probably that was before the StatusEventKind was turned into an interface.